k8s/upgrade/src/bin/upgrade-job/helm/client.rs

```diff
@@ -68,7 +68,7 @@ impl HelmReleaseClientBuilder {
 /// releases.
 #[derive(Clone)]
 pub(crate) struct HelmReleaseClient {
-    namespace: String,
+    pub namespace: String,
```

Comment:
Seems a bit odd to have a pub(crate) struct with a pub method.

Reply:
changed to pub(crate)